### PR TITLE
Show first steps for n00bs.

### PR DESCRIPTION
**Pull request by benlaurie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,4 +7,16 @@ white paper, which in turn could be thought of as an extension and
...
+tree, then:
+
+    % cd <your favourite directory for git repos>
+    % git clone https://github.com/google/trillian.git
+    % ln -s `pwd`/trillian $GOPATH/src/github.com/google  # you may have to make this directory first
+    % cd trillian
+    % go get -d -v ./...
+    % go test -v ./...
+
+If you are with the Go program, then you know what to do.
```
Comment (AlCutter) [inline README.md]: Worth noting that the tests under `storage/mysql` need a mysql instance running to work?
Could point to `.travis.yml` as a reference for how to set it up...

Comment (benlaurie) [inline README.md]: I have updated the destructions...

**Comment by AlCutter** [comment]
Looks like it needs a rebase :/

**Comment by Martin2112** [comment]
Can you rebase and fix the conflicts so we can merge?

**Comment by benlaurie** [comment]
As I mentioned to Al, you really shouldn't be asking your generous contributors to do work you could trivially do yourself.

Anyway, merged and pushed at c9a1d43.

**Comment by Martin2112** [comment]
Thanks. I don't think I can even begin to express our gratitude to our
generous contributors.

Martin

**Pull request closed without merging**